editor/src/components/inspector/inspector.tsx

```diff
@@ -805,6 +805,11 @@ export const InspectorContextProvider = betterReactMemo<{
     } else {
       const elementMetadata = MetadataUtils.getElementByInstancePathMaybe(jsxMetadataKILLME, path)
       if (elementMetadata != null) {
+        if (elementMetadata.computedStyle == null) {
```

Comment:
Interesting... Why do we stop here and not do everything except the line:
```newComputedStyles.push(elementMetadata.computedStyle)```
?

Reply:
maybe I should make the comment above the return statement more clear. If we continue and only skip newComputedStyles, it is very very easy to make a silent error, which 100% means we would have regressions around the computedStyles stuff

Reply:
Ah ok yes please then. It concerns me that this logic is going to live here, but I don't have a better proposal tbh. The only alternative I can think of is to make the metadata a union type to represent if it is the full runtime metadata or not, but I'm not sure that would be worth the huge refactor it would require.

Reply:
I think I'll come back to this later this week, to prevent those 100 extra rerenders